spring-integration-core/src/main/java/org/springframework/integration/dsl/GatewayProxySpec.java

```diff
@@ -43,7 +44,7 @@
  *
```

Comment:
Your name here, please
```suggestion
 * @author Gary Russell
```